posthog/models/ingestion_warnings/sql.py

```diff
@@ -0,0 +1,74 @@
+from django.conf import settings
+
+from posthog.clickhouse.kafka_engine import KAFKA_COLUMNS, kafka_engine
+from posthog.clickhouse.table_engines import Distributed, MergeTreeEngine, ReplicationScheme
+from posthog.kafka_client.topics import KAFKA_INGESTION_WARNINGS
+
+INGESTION_WARNINGS_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} ON CLUSTER '{cluster}'
```

Comment:
I suspect this table won't be too large but might be worth adding a TTL? Something large even would do like a year? Just think that nobody will care about old warnings.

Perhaps overoptimizing though.

Reply:
yup, I think we can just drop the old parts, but not a CH expert

Reply:
Yeah we can always delete data manually - and on Cloud that's no problem. 

For self-hosted users I believe this table's size will be negligible, but wouldn't necessarily hurt to have the TTL

Reply:
While TTL wouldn't hurt I don't see a reason to add it either - maybe we'll change the FE in incompatible ways?

I'd punt on this and add the TTL in a follow-up